subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/CrossVersionPerformanceTestRunner.groovy

```diff
@@ -101,10 +95,6 @@ class CrossVersionPerformanceTestRunner extends PerformanceTestSpec {
         buildMutators.add(buildMutator)
     }
 
-    List<String> getMeasuredBuildOperations() {
```

Comment:
Please leave this method in, it is sometimes used in branches.

Reply:
fixed